### PR TITLE
Update caching.md

### PR DESCRIPTION
**Pull request by thomasr8**
Reworded "Clearing Cache" to advise readers that clearing a cache is not currently supported. I also added a note that users should attempt to reduce the size of the cache whenever possible.

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/caching.md [hunk 1]
```diff
@@ -197,7 +197,9 @@ Caches created via the `save_cache` step are stored for up to 15 days.
 ### Clearing cache
 {: #clearing-cache }
 
-If you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
+Caches currently cannot be cleared; however, if you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
+
+You may create a new cache while still still keeping the "older" cache by incrementing the cache version; however, be aware that you will be creating an additional cache, which will be available for 15 days and will increase your storage. As a general best practice, you should review what is currently being cached and reduce its size as much as possible.
```
Comment (jabsci) [inline jekyll/_cci2/caching.md:202]: Duplicate "still" on line 202

File: jekyll/_cci2/caching.md [hunk 2]
```diff
@@ -197,7 +197,9 @@ Caches created via the `save_cache` step are stored for up to 15 days.
 ### Clearing cache
 {: #clearing-cache }
 
-If you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
+Caches currently cannot be cleared; however, if you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
```
Comment (jabsci) [inline jekyll/_cci2/caching.md:200]: I would potentially reword this to: 
" Caches currently cannot be cleared. If you need to generate a new set of caches because you've updated language or build management tool versions, simply update the cache key, similar to the previous example. 

Updating the cache key on save and restore steps in the `config.yml` file will then generate new sets of caches from that point onwards. Please note that older commits using the previous keys may still generate & save cache for builds, so remember to rebase after the `config.yml` changes when possible."

**Review by jabsci — COMMENTED** [review]
Hey Thomas, thank you for including me on the review! I've left a couple of comments, the first one regarding line 200 is not a deal breaker but the typo on line 202 should be amended before merge.

File: jekyll/_cci2/caching.md [hunk 3]
```diff
@@ -197,7 +197,11 @@ Caches created via the `save_cache` step are stored for up to 15 days.
 ### Clearing cache
 {: #clearing-cache }
 
-If you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
+Caches currently cannot be cleared. If you need to generate a new set of caches because you have updated language or build management tool versions, simply update the cache key, similar to the previous example.
+
+Updating the cache key on save and restore steps in the 'config.yml' file will then generate new sets of caches from that point onwards. Please note that older commits using the previous keys may still generate & save cache for builds, so remember to rebase after the 'config.yml' changes when possible
```
Comment (jabsci) [inline jekyll/_cci2/caching.md:202]: Missing a full-stop at the end: `...changes when possible.`

Comment (thomasr8) [reply jekyll/_cci2/caching.md]: Thanks for the good catch. Fixed.

**Review by jabsci — APPROVED** [review]

File: jekyll/_cci2/caching.md [hunk 4]
```diff
@@ -197,7 +197,11 @@ Caches created via the `save_cache` step are stored for up to 15 days.
 ### Clearing cache
 {: #clearing-cache }
 
-If you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
+Caches currently cannot be cleared. If you need to generate a new set of caches because you have updated language or build management tool versions, simply update the cache key, similar to the previous example.
+
+Updating the cache key on save and restore steps in the 'config.yml' file will then generate new sets of caches from that point onwards. Please note that older commits using the previous keys may still generate & save cache for builds, so remember to rebase after the 'config.yml' changes when possible.
```
Comment (rosieyohannan) [inline jekyll/_cci2/caching.md:202]:
```suggestion
Updating the cache key on save and restore steps in your '.circleci/config.yml' file will then generate new sets of caches from that point onwards. Please note that older commits using the previous keys may still generate and save cache, so it is recommended that you rebase after the 'config.yml' changes when possible.
```

File: jekyll/_cci2/caching.md [hunk 5]
```diff
@@ -197,7 +197,11 @@ Caches created via the `save_cache` step are stored for up to 15 days.
...
+
+You may create a new cache while still keeping the "older" cache by incrementing the cache version; however, be aware that you will be creating an additional cache, which will be available for 15 days and will increase your storage. As a general best practice, you should review what is currently being cached and reduce its size as much as possible.
```
Comment (rosieyohannan) [inline jekyll/_cci2/caching.md:204]:
```suggestion
If you create a new cache by incrementing the cache version, the "older" cache is still stored. It is important to be aware that you are creating an additional cache, which will be available for 15 days. This method will increase your storage usage. As a general best practice, you should review what is currently being cached and reduce your storage usage as much as possible.
```

File: jekyll/_cci2/caching.md [hunk 6]
```diff
@@ -197,7 +197,11 @@ Caches created via the `save_cache` step are stored for up to 15 days.
 ### Clearing cache
 {: #clearing-cache }
 
-If you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
+Caches currently cannot be cleared. If you need to generate a new set of caches because you have updated language or build management tool versions, simply update the cache key, similar to the previous example.
```
Comment (rosieyohannan) [inline jekyll/_cci2/caching.md:200]:
```suggestion
Caches cannot be cleared. If you need to generate a new set of caches because you have updated language or build management tool versions, simply update the cache key, similar to the previous example.
```

Comment (rosieyohannan) [inline jekyll/_cci2/caching.md]:
```suggestion
Caches currently cannot be cleared. If you need to generate a new set of caches you can update the cache key, similar to the previous example. You may wish to do this if you have updated language or build management tool versions.
```

**Review by rosieyohannan — APPROVED** [review]
